tests/acceptance/guid-node/metadata-test.ts

```diff
@@ -197,30 +197,6 @@ module('Acceptance | guid-node/metadata', hooks => {
         await percySnapshot(assert);
     });
 
-    test('Error handling: metadata', async function(this: TestContext, assert) {
```

Comment:
Not sure what's going on with this test. It passes in the browser and command line, but fails when the tests are run in parallel. When run in parallel, it seems it can't find element `#toast-container`.

Reply:
Would it be better to `skip` this and make a ticket to try to figure out what's going on?